content/get.md

```diff
@@ -13,51 +13,50 @@ Currently (June 2021), we support machine configurations for:
 -   saga (sigma2, Norway)
 -   fram (sigma2, Norway)
 
-If your machine is not in the list and you would like us to support it,
+If your machine is not on the list and you would like us to support it,
 please contact us.
 
 ## How to get CTSM (for users)
 A user is a person that runs CTSM without modifying the source code. To get the FATES EMERALD platform version, CLONE from NordicESM hub
 
     [~/HOME]$ git clone -b release-clm5.0 https://github.com/NordicESMhub/ctsm.git ${HOME}/ctsm_fates_emerald
 
-In this example we are checking out the release-clm5.0 tag and create a
+In this example, we are checking out the release-clm5.0 tag and create a
 new local branch (recommended). The destination of the checkout is a
 directory (e.g. ctsm\_fates\_emerald) in our home directory.
```

Comment:
Considering this is markdown - maybe add hyperlinks to references explaining the git terms?

Reply:
I have made a new part called `prerequisites.md` (currently in #5) which is linking to git tutorials and other stuff you need to know. I can add a link from that page if you think that's clearer?